presto-common/pom.xml

```diff
@@ -115,4 +115,112 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>java21PrestoCommon</id>
```

Comment:
just curious why the build profiles are named differently for every module vs. all having a profile called java21 and javaLessThan21?